src/validation.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  type DscvrFramesRequest,
+  type DscvrValidationResponse,
+  dscvrClientProtocolPrefix,
```

Comment:
I suggest moving `dscvrClientProtocolPrefix` to `src/constants.ts` instead of `src/types.ts` as it's an `const` that is being exported rather than a `type`

Reply:
Done